internal/output/templates.go

```diff
@@ -365,3 +356,56 @@ This comment will be replaced when the cost estimate changes
 </sub>
 {{- end}}
 `
+
+var CommentMarkdownTemplate = `
+{{- define "summaryRow"}}
+| {{ truncateMiddle .Name 64 "..." }} | {{ formatCost .PastCost }} | {{ formatCost .Cost }} | {{ formatCostChange .PastCost .Cost }} |
+{{- end }}
+{{- define "totalRow"}}
+| **{{ truncateMiddle .Name 64 "..." }}** | **{{ formatCost .PastCost }}** | **{{ formatCost .Cost }}** | **{{ formatCostChange .PastCost .Cost }}** |
+{{- end }}
+## 💰 Infracost estimate: **{{ formatCostChangeSentence .Root.Currency .Root.PastTotalMonthlyCost .Root.TotalMonthlyCost }}**
+
+| **Project** | **Previous** | **New** | **Diff** |
+| ----------- | -----------: | ------: | -------- |
+
+{{- if gt (len .Root.Projects) 1  }}
+  {{- range .Root.Projects }}
+    {{- if hasDiff . }}
+      {{- template "summaryRow" dict "Name" .Name "PastCost" .PastBreakdown.TotalMonthlyCost "Cost" .Breakdown.TotalMonthlyCost  }}
+    {{- end }}
+  {{- end }}
+  {{- template "totalRow" dict "Name" "All projects" "PastCost" .Root.PastTotalMonthlyCost "Cost" .Root.TotalMonthlyCost  }}
+
+  {{- if eq .SkippedProjectCount 1 }}
+
+1 project has no cost estimate changes.
+  {{- else if gt .SkippedProjectCount 0 }}
+
+{{ .SkippedProjectCount }} projects have no cost estimate changes.
+  {{- end }}
+{{- else }}
+  {{- range .Root.Projects }}
+    {{- template "summaryRow" dict "Name" .Name "PastCost" .PastBreakdown.TotalMonthlyCost "Cost" .Breakdown.TotalMonthlyCost  }}
+  {{- end }}
+{{- end }}
+
+**Infracost output:**
+
+` + "```" /* can't escape backticks */ + `
+{{ .DiffOutput }}
+` + "```" /* can't escape backticks */ + `
+
+{{- if .MarkdownOptions.WillUpdate }}
+
+This comment will be updated when the cost estimate changes.
+{{- end }}
+{{- if .MarkdownOptions.WillReplace }}
+
+This comment will be replaced when the cost estimate changes
+{{- end }}
+{{- if .MarkdownOptions.IncludeFeedbackLink }}
+
+Is this comment useful? [Yes](https://www.infracost.io/feedback/submit/?value=yes), [No](https://www.infracost.io/feedback/submit/?value=no)
+{{- end }}
+`
```

Comment:
@alikhajeh1 Here's the Bitbucket comment structure. Does it look okay?

Reply:
Do you have a screenshot or a link to an example comment?

Reply:
We could default tag to "", and if they set it then the last line could be _(generated by Infracost, tag: X)_.

Reply:
Looks great! I'd keep it as is since there is a use-case for matrix builds wanting to customize the tag.

Reply:
@aliscott Great idea! I'll do that, thanks! 👍

Reply:
Updated: https://github.com/infracost/infracost/pull/1363/files#diff-b209df5ca9c5860317f0907a53d70e6889cbd10f3e352ae6bd8674becb38fe75R162